lib/nghttp2_hd.c

```diff
@@ -1503,6 +1503,38 @@ ssize_t nghttp2_hd_deflate_hd(nghttp2_hd_deflater *deflater, uint8_t *buf,
   return (ssize_t)buflen;
 }
 
+ssize_t nghttp2_hd_deflate_hd_vect(nghttp2_hd_deflater *deflater, uint8_t **bufsin,
+                              size_t inlen, size_t buflen, const nghttp2_nv *nv,
+                              size_t nvlen) {
```

Comment:
We have standard struct io_vec, so perhaps, nghttp2_hd_deflate_hd_vec() sounds more like C?